### PR TITLE
Links support for mediaccc and shortened invidious

### LINKED ISSUE #3079: Shortened invidio.us link support
NewPipe supports shortened yt links and invidio.us links, but not shortened invidio links.

Example:
> https://invidio.us/X8avbciUP3c

will lead one to the browser, and only then to NewPipe, whereas
> https://invidio.us/watch?v=X8avbciUP3c

leads to NewPipe directly.

Can this please be implemented=

### LINKED ISSUE #3080: Mediaccc links support
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.

https://media.ccc.de/v/oscal19-12-the_hairy_issue_of_end_to_end_encrypted_instant_messaging
It should open as a video. Not supported by extractor, we have "link not supported" toast.

https://media.ccc.de/b/oscal19
It should open as conference (or channel for a youtube comparison). Not supported by extractor, we have "link not supported" toast.

https://media.ccc.de/c/oc it should open as conference. It's supported by the extractor, but not by the app (try with sharing it into newpipe).

### PR DESCRIPTION
**Pull request by B0pol**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

Relies on TeamNewPipe/NewPipeExtractor#256
Fixes #3079 
Fixes #3080 

### Test
APK:  [2020-02-12_links#3090.zip](https://github.com/TeamNewPipe/NewPipe/files/4194117/2020-02-12_links.3090.zip)

Links: 
+ https://media.ccc.de/v/oscal19-12-the_hairy_issue_of_end_to_end_encrypted_instant_messaging
+ https://media.ccc.de/b/oscal19
+ https://media.ccc.de/c/oc
+ https://invidio.us/-cdveCh1kQk

### CODE REVIEW

**Comment by B0pol** [comment]
About #3087
I worked on the extractor yesterday, and finished yesterday.
I also refactored tests. 
The extractor PR isn't done for #3087, and there is no support for short invidious links.

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -246,6 +246,7 @@
                 <!-- video prefix -->
                 <data android:pathPrefix="/embed/"/>
                 <data android:pathPrefix="/watch"/>
+                <data android:pathPrefix="/"/>
```
Comment (wb9688) [inline app/src/main/AndroidManifest.xml]: If you're doing that, you should remove the 2 lines above, since they also start with a / ;)

Comment (B0pol) [reply app/src/main/AndroidManifest.xml]: and also the lines below. thank you

**Comment by TobiGr** [comment]
Can you please update the extractor version?

**Pull request merged**